### DISCUSSION TITLE
About CPU voltage reduction and overclocking

### DISCUSSION THREAD

**Discussion by hgjhguy** [BartoszCichecki/LenovoLegionToolkit#1330] [Ideas]
Thank you. May I make a small suggestion to add a CPU voltage reduction and overclocking function?

**Comment by BartoszCichecki** [comment]
CPU OC/UV will not be added to LLT as there already are very good and light apps for that like UXTU or ThrottleStop.

**Reply by Rar9** [reply]
Which is recommended UXTU or ThrottleStop?

Is there a guide so LLT and CPU OC/UV won't conflict and  work nicely to prolong battery  running time by undervolting?

**Reply by Kucerarj** [reply]
I have a Legion 7i gen7 and what I am noticing is that I can't overclock with Throttlestop. I have to go into Vantage, and set the multipliers and then turn on overclock. Undervolting with Throttlestop works and I can set all p-core multipliers to 47 and e-cores to 34 and that works, but can't go above those unless done through vantage. And right now I can't even get XTU to start, it gets stuck on the splash screen. I did a clean install even and XTU still has the same issue. I've been overclocking for over 25 years, so familiar with the process. Thanks

**Reply by XyKER1** [reply]
> Which is recommended UXTU or ThrottleStop?
> 
> Is there a guide so LLT and CPU OC/UV won't conflict and work nicely to prolong battery running time by undervolting?

I would recommend Throttlestop since it's been around for years and has every possible option, while UXTU only started supporting Intel recently afaik.

Throttlestop should not conflict with Legion Toolkit except if you modify the power limits in Throttlestop and check the Sync MMIO or lock box next to MMIO, which will override any power limits set by Lenovo Legion Toolkit.

**Reply by Kucerarj** [reply]
XTU is made by Intel and has been around for years. Seems like with newer laptops, Throttlestop cannot overclock. I fixed my issue with XTU not starting. Lenovo released a 46 bios for the Legion 7i gen7 that was pulled after being out for a day. I flashed back to 45 and XTU works for overclocking. I have used Throttlestop for over 10 years, but looks like development has pretty much stopped.

**Reply by XyKER1** [reply]
@Rar9 asked about UXTU, XTU I can recommend although I would still prefer Throttlestop. My Legion Pro 7i Gen 8 has no issues overclocking so I can't say anything negative about it. Unclewebb has not stopped development but there is nothing else he can add. When new CPUs are released, he always updates it promptly with support for them.

**Reply by Kucerarj** [reply]
> @Rar9 asked about UXTU, XTU I can recommend although I would still prefer Throttlestop. My Legion Pro 7i Gen 8 has no issues overclocking so I can't say anything negative about it. Unclewebb has not stopped development but there is nothing else he can add. When new CPUs are released, he always updates it promptly with support for them.

I didn't realize there was a UXTU, I've never used AMD procs except in servers. Just thought the "U" was a typo. I've read that Unclewebb doesn't have newer CPUs to test with. I've used Throttlestop for a long time, and a lot of that stuff from back then is still there. Would be nice to see it cleaned up.

**Reply by XyKER1** [reply]
It could use a nicer UI, but there are a LOT of useful features in there. I would rather have more options than less personally, and I've made of use of probably at least 50 different features of Throttlestop in the past.
